crypto/rsa/rsa_impl.c

```diff
@@ -646,11 +635,9 @@ static int mod_exp(BIGNUM *r0, const BIGNUM *I, RSA *rsa, BN_CTX *ctx) {
     goto err;
   }
 
-  /* Turn BN_FLG_CONSTTIME flag on before division operation */
-  pr1 = &local_r1;
-  BN_with_flags(pr1, r1, BN_FLG_CONSTTIME);
+  assert(BN_get_flags(r1, BN_FLG_CONSTTIME));
```

Comment:
NIT: It doesn't matter much, but might as well assert `rsa->p` has the consttime flag, instead of assertint that `r1` has it, to be consistent with the above assertions.